src/components/duelling-picklist/picklist-item/picklist-item.test.tsx

```diff
@@ -79,26 +78,17 @@ test("the context callback is called with the expected arguments when isLastItem
   expect(setElementToFocus).toHaveBeenCalledWith(0, 1, undefined);
 });
 
-// test purely for coverage - the `locked` prop is tested in both Chromatic and Playwright [although not with the focusRedesignOptOut flag]
+// test purely for coverage - the `locked` prop is tested in both Chromatic and Playwright
 test("should render a locked icon with expected styling when the `locked` prop is set and the focusRedesignOptOut flag is set", () => {
```

Comment:
**suggestion**: could we remove this part from the description too for completeness, given the flag no longer exists? :) 

```suggestion
test("should render a locked icon with expected styling when the `locked` prop is set", () => {
```